sky/clouds/service_catalog/data_fetchers/fetch_gcp.py

```diff
@@ -458,11 +563,22 @@ def _get_tpus() -> 'pd.DataFrame':
 
 
 # TODO: the TPUs fetched fails to contain us-east1
-def get_tpu_df(skus: List[Dict[str, Any]]) -> 'pd.DataFrame':
+def get_tpu_df(gce_skus: List[Dict[str, Any]],
+               tpu_skus: List[Dict[str, Any]]) -> 'pd.DataFrame':
     df = _get_tpus()
     if df.empty:
         return df
 
+    def _get_tpu_str(tpu_version: str) -> str:
```

Comment:
For better readability, we may want to change the function name a bit.
```suggestion
    def _get_tpu_description_str(tpu_version: str) -> str:
```

Reply:
Good point! Done. Thanks!